src/externalized/DefaultContentManager.cc

```diff
@@ -1459,6 +1471,15 @@ const std::vector<const MercProfile*>& DefaultContentManager::listMercProfiles()
 	return m_mercProfiles;
 }
 
+void DefaultContentManager::resetMercProfileStructs() const
+{
+	for (size_t i = 0; i < NUM_PROFILES; i++) {
+		if (m_mercStructs[i] != nullptr) {
```

Comment:
Can any of the mercStructs pointers really be null? If yes, you should add an else branch to at least set gMercProfiles[i] to a default value.

Reply:
Yes, they can. I just put a safeguard against it in `loadMercsData()`